src/System.Text.Json/tests/Serialization/BlogPost.cs

```diff
@@ -0,0 +1,821 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+
+namespace System.Text.Json.Serialization.Tests
```

Comment:
`Tag` and `Post` and `Metadata` and such are very generic terms, and might be unfortunate to mix into this namespace.

Consider making this file a sub-namespace, like `System.Text.Json.Serialization.Tests.Schemas.BlogPost`